test/app-tap/exports.test.lua

```diff
@@ -52,6 +52,8 @@ local check_symbols = {
     'tnt_mp_sizeof_error',
     'tnt_mp_sizeof_decimal',
     'tnt_mp_sizeof_uuid',
+    'tnt_tx_flush',
+    'tnt_tx_push',
```

Comment:
I think this belongs in the previous patch.

Reply:
Indeed, thanks for the notice, done.